Policies/Kubernetes/selinux/rego/src_test.rego

```diff
@@ -0,0 +1,203 @@
+package k8sazureselinux
+
+test_input_seLinux_options_allowed_in_list {
+    input := { "review": input_review, "parameters": input_parameters_in_list}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_input_seLinux_options_allowed_in_list_subset {
+    input := { "review": input_review, "parameters": input_parameters_in_list_subset}
+    results := violation with input as input
+    count(results) == 0
+}
+
+test_input_seLinux_option_not_allowed_not_in_list {
+    input := { "review": input_review, "parameters": input_parameters_not_in_list}
+    results := violation with input as input
+    count(results) == 1
+}
+
+test_input_seLinux_options_empty {
+    input := { "review": input_review, "parameters": input_parameters_empty}
```

Comment:
As input_parameters_empty is a valid case and it means SELinux properties should not be defined, we need to specially mention this in parameter description.